modules/web/js/ballerina/diagram2/views/default/components/decorators/panel-decorator.jsx

```diff
@@ -81,7 +81,14 @@ class PanelDecorator extends React.Component {
     }
 
     onTitleClick() {
-        if (this.props.model.getKind() === 'Function' && this.props.model.getName().value === 'main') {
+        const model = this.props.model;
+        if (model.getKind() === 'Function'
+            && model.getName().value === 'main'
```

Comment:
Can we move this logic to tree util and use them in other places. ie isMainFunction ...